<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="RUsernamelineEdit_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="RUsernamelabel_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/IMAGE/1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="RUsernamelineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="REmaillabel_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>420</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="RPasswordlabel_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MBACK">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>460</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Bebas&quot;;
color: white;
background-color: rgb(0, 0, 0);



</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MREGISTER">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Bebas&quot;;
color: white;
background-color: rgb(0, 0, 0);



</string>
   </property>
   <property name="text">
    <string>REGISTER</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RPasswordlineEdit_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="RLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>RUsernamelineEdit_2</zorder>
  <zorder>RUsernamelabel_2</zorder>
  <zorder>RUsernamelineEdit</zorder>
  <zorder>REmaillabel_3</zorder>
  <zorder>RPasswordlabel_3</zorder>
  <zorder>MBACK</zorder>
  <zorder>MREGISTER</zorder>
  <zorder>RPasswordlineEdit_2</zorder>
  <zorder>RLabel</zorder>
 </widget>
 <resources>
  <include location="IMAGES.qrc"/>
 </resources>
 <connections/>
</ui>
